### PR TITLE
Formatting, fixing typos

### PR DESCRIPTION
**Pull request by jknagin**
# Summary
* PEP 8 formatting
* Fixes typos
* Replaces sort in `findBestWord` with search for minimum

### CODE REVIEW

File: compute.py [hunk 1]
```diff
@@ -42,20 +45,27 @@ def compute(cls, query: Word, secret: Word):
             greenIdxSet = secretIdxSet.intersection(queryIdxSet)
             for greenIdx in greenIdxSet:
                 filter[greenIdx] = Color.GREEN
+<<<<<<< HEAD
```
Comment (chandler-g-chen) [inline compute.py]: conflict markers?

Comment (jknagin) [reply compute.py]: Oh I guess I missed this because you pushed to main right around when I made my branch

File: compute.py [hunk 2]
```diff
@@ -80,25 +86,23 @@ def filterFromInput() -> Filter:
         enums.append(strToEnum[c])
     return Filter(*enums)
 
+
 with open("sgb-words.txt") as f:
     wordBank = [Word(s) for s in sorted(f.read().splitlines())]
 
-# assert Filter.compute(Word("oooll"), Word("llool")) == Filter(Color.YELLOW,Color.GRAY,Color.GREEN,Color.YELLOW,Color.GREEN)
-
-# Generated from findBestWord(wordBank, wordBank)[0][1], cached here since it takes a while. Need to rerun for new wordbank
```
Comment (chandler-g-chen) [inline compute.py:88]: I want to keep this doc

**Review by chandler-g-chen — CHANGES_REQUESTED** [review]

File: compute.py (hunk 2 above)
Comment (jknagin) [reply compute.py:88]: Okay

**Pull request merged**